README.md

```diff
@@ -139,6 +139,18 @@ You can run these tests as below:
 
 **_Current restriction - max 2 web browsers_**
 
+### Running Multi-User scenarios in Mobile Emulation View of Web Browsers
+
+The framework now supports launching web browsers in Mobile Emulation View.
+Instead of passing simple browser names in steps like 'chrome', 'firefox', etc, 
+you can pass more informative parameters like 'chrome-mobile1', 'safari-mobile2', etc
+
+These informative parameters refer to Browser Config json file for actual device names
+
+Example:
+
+![mobile-emulation-on-web-browser.png](mobile-emulation-on-web-browser.png)
```

Comment:
this png seems to be missed for checking

Reply:
added now (previously ignored due to gitignore)